src/IO/GEHDF5Wrapper.cxx

```diff
@@ -283,18 +284,29 @@ Succeeded GEHDF5Wrapper::initialise_singles_data(const std::string &path)
 {
     if(path.size() == 0)
     {
-        if(is_signa)
+        if(is_signa) //AB todo "is_RDF9"
         {
-            m_address = "/Singles/CrystalSingles/sample";
-            //! \todo Get these numbers from the HDF5 file
-            {
-                m_NX_SUB = 45;    // hyperslab dimensions
-                m_NY_SUB = 448;
-                m_NZ_SUB = 1;
-                m_NX = 45;        // output buffer dimensions
-                m_NY = 448;
-                m_NZ = 1;
-            }
+            m_address = "/Singles/CrystalSingles/sample1"; //AB: todo. '/slms/inm/research/moco/PhantomData_Orsay/R00037/raw/rdf.0' had ./sample1 but ./sample was written here. ?? typo?
+            // Get the DataSpace (metadata) corresponding to the DataSet that we want to read
```

Comment:
The m_address here reads the samples accross the time of scan to collect singles per crystal. The m_address should be:
m_address = "/Singles/CrystalSingles/sample";